docs/company-kata/slides.md

```diff
@@ -596,7 +596,7 @@ ImmutableList<Integer> immutableList =
 ImmutableList<Integer> immutableList2 =
   Lists.immutable.of(1, 2, 3);
 
-Verify.assertInstanceOf(ImmutableTripletonList.class, immutableList);
+Verify.assertInstanceOf(ImmutableList.class, immutableList);
```

Comment:
Hi @mydeveloperplanet, the test for `ImmutableTripletonList.class` is correct here. The test for `ImmutableList.class` is unnecessary because otherwise the code would not compile on line 593.

Reply:
Hi @donraab , I agree that my change is useless, but I do not understand why the original assert was correct: `ImmutableTripletonList` cannot be accessed from outside the package. I tried to run the code by creating a new test in the `Exercise2Test.java `file in order to verify how this should work. So, any idea what I am doing wrong?

Reply:
Hi @mydeveloperplanet, I think the idea in the slide was to demonstrate if the test was implemented in Eclipse Collections in the package it would test to validate that you get back specifically `ImmutableTripletonList` even though a developer can never actually reference this class in their own code..